modules/gdscript/gdscript_editor.cpp

```diff
@@ -389,7 +390,22 @@ void GDScriptLanguage::debug_get_globals(List<String> *p_globals, List<Variant>
 }
 
 String GDScriptLanguage::debug_parse_stack_level_expression(int p_level, const String &p_expression, int p_max_subitems, int p_max_depth) {
-	return "";
+    List<String> names;
+    List<Variant> values;
+    debug_get_stack_level_locals(p_level, &names, &values, p_max_subitems, p_max_depth);
+
+    Vector<String> name_vector;
+    for (int i = 0; i < names.size(); ++i)
```

Comment:
There are some style issues:
- You should tabs for indentation
- There should be braces around `for` and `if` scopes, even if single line.
*Note:* Please make modifications by _amending_ the commit, and *force pushing* the changes, so that it stays as a single commit (see [PR workflow](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#modifying-a-pull-request)).

Reply:
@akien-mga Thanks for reviewing. Should have learn the coding style (for the first PR).

Feedback addressed.